src/components/Tabs/index.tsx

```diff
@@ -7,7 +7,7 @@ type TabsProps = {
     onSelect: (tab: Tab) => void;
     activeTab: Tab;
 };
-function changeName(name: string) {
+export function changeName(name: string) {
```

Comment:
Suggestion: if this function is being used in other components, you can move it to a util file

Reply:
resolved